packages/preferences/src/store/actions.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Returns an action object used in signalling that a feature should be toggled.
+ *
+ * @param {string} scope       The feature scope (e.g. core/edit-post).
+ * @param {string} featureName The feature name.
+ */
+export function toggleFeature( scope, featureName ) {
```

Comment:
In the future I imagine we'll have not only boolean preferences in this store but also other types. Probably an 'enum' style preference would be next.

I considered whether there should just be one function for setting any kind of preference, but I think a `toggle` style function is quite a nice convenience compared to a more freeform `setPreferenceValue`, so I've kept this API similar to the way it is now.

Renaming it from 'feature' is something we might consider though.